_CONTRIBUTING.md

```diff
@@ -4,11 +4,20 @@
 - [Laravel](https://laravel.com/)
 - [Materialize](https://materializecss.com/)
 
+# GitHub, Git, Branching
+
+- We organise our work around issues. Issues for structural changes should be discussed beforehand.
+- We make a new branch for every issue (or connected issues).
+- We open pull requests to the "development" branch.
+- We regularly update the "deployment" branch with changes on "development".
```

Comment:
We could probably mention `staging`. I usually merge new changes to `staging`, test it out, and then deploy it to production by merging new changes to `deployment`

Reply:
Do we have a staging server as well?

Reply:
Yep, I created a staging server a couple of weeks ago